pkgs/tools/networking/mdk3/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, libnl, libpcap, pkg-config }:
+
+stdenv.mkDerivation {
+  pname = "mdk3";
+  version = "unstable-2015-08-04";
+
+  src = fetchFromGitHub {
+    owner = "aircrack-ng";
+    repo = "mdk3";
+    rev = "3be47e2e02d311e6746bd6d38c73ec957300aba0";
+    sha256 = "sha256-gh8tq3CxYb+qpeqxvKnD3d+qD6uPsBeXSkr42gn7Puc=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libnl libpcap ];
+
+  installFlags = [ "PREFIX=$(out)" "SBINDIR=$(PREFIX)/bin" ];
+
+  meta = with lib; {
+    description = "A tool that injects data into wireless networks";
+    homepage = "https://github.com/aircrack-ng/mdk3";
+    maintainers = with maintainers; [ fortuneteller2k ];
+    license = licenses.gpl3Only;
```

Looks like [GPLv2+](https://github.com/aircrack-ng/mdk3/blob/master/src/osdep/dummy.c#L6-L9).

```suggestion
    license = licenses.gpl2Plus;
```